en/orm/validation.rst

```diff
@@ -377,6 +377,16 @@ can help provide a nicer user experience. Because of this CakePHP includes an
 The fields to check existence against in the related table must be part of the
 primary key.
 
+.. versionadded:: 3.2.11
+   You may now enforce existsIn to pass on parts of your composite foreign key to be null.
+
+   // The primary composite key of Nodes is (id, site_id).
```

Comment:
because its the Association name (Nodes, ParentNodes) I can change it to nodes. no problem.